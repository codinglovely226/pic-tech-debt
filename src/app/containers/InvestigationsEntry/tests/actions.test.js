import {
  hideInvestigationsViewResultModal,
  removeInvestigationsViewResultAttachment,
  resetInvestigationsViewRequest,
  resetInvestigationsViewResult,
  showInvestigationsViewResultModal,
  updateInvestigationsRequests,
  updateInvestigationsResults,
  updateInvestigationsViewRequest,
  updateInvestigationsViewResultAttachments,
  updateInvestigationsViewResultDescription,
  updateInvestigationsViewResultNotes,
} from '../actions/case';

describe('InvestigationsEntry action creators', () => {
  it('hideInvestigationsViewResultModal', () => {
    expect(hideInvestigationsViewResultModal()).toMatchSnapshot();
  });
  it('removeInvestigationsViewResultAttachment', () => {
    expect(removeInvestigationsViewResultAttachment(1)).toMatchSnapshot();
  });
  it('resetInvestigationsViewRequest', () => {
    expect(resetInvestigationsViewRequest()).toMatchSnapshot();
  });
  it('resetInvestigationsViewResult', () => {
    expect(resetInvestigationsViewResult()).toMatchSnapshot();
  });
  it('showInvestigationsViewResultModal', () => {
    expect(showInvestigationsViewResultModal(1)).toMatchSnapshot();
  });
  it('updateInvestigationsRequests', () => {
    expect(updateInvestigationsRequests(1, 0, 2)).toMatchSnapshot();
  });
  it('updateInvestigationsResults', () => {
    expect(updateInvestigationsResults(1, 0, 3)).toMatchSnapshot();
  });
  it('updateInvestigationsViewRequest', () => {
    expect(updateInvestigationsViewRequest('Test Content')).toMatchSnapshot();
  });
  it('updateInvestigationsViewResultAttachments', () => {
    expect(updateInvestigationsViewResultAttachments([1, 2, 3])).toMatchSnapshot();
  });
  it('updateInvestigationsViewResultDescription', () => {
    expect(updateInvestigationsViewResultDescription('Test description')).toMatchSnapshot();
  });
  it('updateInvestigationsViewResultNotes', () => {
    expect(updateInvestigationsViewResultNotes('Test content')).toMatchSnapshot();
  });
});
