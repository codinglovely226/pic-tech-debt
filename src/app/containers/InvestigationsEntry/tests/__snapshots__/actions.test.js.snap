exports[`InvestigationsEntry action creators hideInvestigationsViewResultModal 1`] = `
Object {
  "type": "pic/InvestigationsEntry/HIDE_INVESTIGATIONS_VIEW_RESULT_MODAL",
}
`;

exports[`InvestigationsEntry action creators removeInvestigationsViewResultAttachment 1`] = `
Object {
  "index": 1,
  "type": "pic/InvestigationsEntry/REMOVE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENT",
}
`;

exports[`InvestigationsEntry action creators resetInvestigationsViewRequest 1`] = `
Object {
  "type": "pic/InvestigationsEntry/RESET_INVESTIGATIONS_VIEW_REQUEST",
}
`;

exports[`InvestigationsEntry action creators resetInvestigationsViewResult 1`] = `
Object {
  "type": "pic/InvestigationsEntry/RESET_INVESTIGATIONS_VIEW_RESULT",
}
`;

exports[`InvestigationsEntry action creators showInvestigationsViewResultModal 1`] = `
Object {
  "requestIx": 1,
  "type": "pic/InvestigationsEntry/SHOW_INVESTIGATIONS_VIEW_RESULT_MODAL",
}
`;

exports[`InvestigationsEntry action creators updateInvestigationsRequests 1`] = `
Object {
  "id": 1,
  "noteId": 2,
  "requestIx": 0,
  "type": "pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_REQUESTS",
}
`;

exports[`InvestigationsEntry action creators updateInvestigationsResults 1`] = `
Object {
  "id": 1,
  "noteId": 3,
  "requestIx": 0,
  "type": "pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_RESULTS",
}
`;

exports[`InvestigationsEntry action creators updateInvestigationsViewRequest 1`] = `
Object {
  "content": "Test Content",
  "type": "pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_REQUEST",
}
`;

exports[`InvestigationsEntry action creators updateInvestigationsViewResultAttachments 1`] = `
Object {
  "attachments": Array [
    1,
    2,
    3,
  ],
  "type": "pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENTS",
}
`;

exports[`InvestigationsEntry action creators updateInvestigationsViewResultDescription 1`] = `
Object {
  "description": "Test description",
  "type": "pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_RESULT_DESCRIPTION",
}
`;

exports[`InvestigationsEntry action creators updateInvestigationsViewResultNotes 1`] = `
Object {
  "content": "Test content",
  "type": "pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_RESULT_NOTES",
}
`;
